### DISCUSSION TITLE
[Feature] transfer the entire album to another user (not sharing)

### DISCUSSION THREAD

**Discussion by LoginByCall** [immich-app/immich#13334] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add the ability to transfer an entire album to another user (not sharing). That is, user1 sends a transfer request, and user2 must accept the request. After that, the album is copied from user1's storage to user2's storage and deleted from user1's storage.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by SelfHostedLife** [comment]
Yes, please.
And per photo/video would be nice too, if possible.
(I have a _lot_ of photos on my account from before my partner decided to also use immich, after being impressed)

**Reply by eerison** [reply]
Yep sometimes I don't have albums, just photos/videos that I want to transfer

**Comment by disgustipated** [comment]
I feel like "and deleted from user1's storage." should be an option in the transfer dialog

**Comment by FilippoCeffaSIE** [comment]
This would also make my life easier.
I built a system that auto creates albums based on folder structure, but also needs to support user created albums. The lack of freedom in transferring ownership makes the design more complicated than it could be.
